src/db.c

```diff
@@ -1375,6 +1379,25 @@ int db_col_attr_set(struct db_filter_col *col,
 			rc = -EOPNOTSUPP;
 		}
 		break;
+	case SCMP_FLTATR_CTL_OPTIMIZE:
+		switch (value) {
+		case 0:
+			/* unsupported */
+			rc = -EOPNOTSUPP;
+			break;
+		case 1:
```

Comment:
I think we can probably simplify this:

```
switch (value) {
case 1:
case 2:
        /* no comment needed here because we want you to look in seccomp.h */
        col->attr.optimize = value;
        break;
default:
        rc = -EOPNOTSUP;
        break;
}
```